Xamarin.Forms.Platform.iOS/Renderers/FormattedStringExtensions.cs

```diff
@@ -63,14 +63,47 @@ internal static NSAttributedString ToAttributed(this Span span, Element owner, C
 			if (text == null)
 				return null;
 
-			NSMutableParagraphStyle style = null;
+			NSMutableParagraphStyle style = new NSMutableParagraphStyle();
 			lineHeight = span.LineHeight >= 0 ? span.LineHeight : lineHeight;
 			if (lineHeight >= 0)
 			{
-				style = new NSMutableParagraphStyle();
 				style.LineHeightMultiple = new nfloat(lineHeight);
 			}
 
+#if __MOBILE__
```

Comment:
We can simplify this by adding `using UITextAlignment = AppKit.NSTextAlignment;` inside the `#else` definition at the top of the file, I think. Then we only need this code written once without the `#if` here.

Reply:
@samhouts Hi, I have done the changes as requested. Can you please have another check, thanks.

Reply:
That was fast! Thanks!